corehq/apps/accounting/tasks.py

```diff
@@ -227,6 +235,19 @@ def send_subscription_reminder_emails(num_days, exclude_trials=True):
             subscription.send_ending_reminder_email()
 
 
+def send_subscription_reminder_emails_dimagi_contact(num_days):
+    today = datetime.date.today()
+    date_in_n_days = today + datetime.timedelta(days=num_days)
+    ending_subscriptions = (Subscription.objects
+                            .filter(is_active=True)
+                            .filter(date_end=date_in_n_days)
```

Comment:
if we think it's better to send reminders twice than not at all you can add the `acks_late` option to the task to prevent it from being cleared until after it successfully completes.

http://celery.readthedocs.org/en/latest/userguide/tasks.html